generator.nl.md

```diff
@@ -0,0 +1,341 @@
+---
+# Translation instructions are after the "#" character in this first section. (They are comments that do not show up in the web page.)
+title: Genereer een Toegankelijkheidsverklaring # Do not translate "title:". Do translate the text after "title:".
+lang: nl   # Change "en" to the translated language shortcode from https://www.iana.org/assignments/language-subtag-registry/language-subtag-registry
+last_updated: 2021-02-10   # Put the date of this translation YYYY-MM-DD (with month in the middle)
+translators: #Add one -name: line for every translator
+  - name: "Bas de Bruin"
+  - name: "Eric Velleman"
+contributors: #Add one -name: line for every contributor 
+  - name: Shadi Abou-Zahra
+  - name: Eric Velleman
+  - name: Sanne Eendebak
+  - name: Roel Antonisse
+  - name: Leon Baauw
+  - name: Bas de Bruin
+permalink: /planning/statements/generator/nl/   # Add the language shortcode to the end; for example /planning/statements/generator/nl/
```

Comment:
I think this should be:
> permalink: /planning/statements/generator/nl    # Add the language shortcode to the end - with no slash at the end; for example /planning/statements/generator/nl

source: https://wai-website-theme.netlify.app/writing/frontmatter/#permalink

Please let me know if you think otherwise -- or if it works with slash at end. (I'm pretty sure previous ones that I did with the slash at the end did not work.) Thanks.